package.json

```diff
@@ -1,13 +1,14 @@
 {
   "private": true,
   "devDependencies": {
-    "@babel/cli": "^7.15.7",
-    "@babel/core": "^7.15.5",
-    "@babel/preset-env": "^7.15.6",
+    "@babel/cli": "^7.16.0",
+    "@babel/core": "^7.16.0",
+    "@babel/preset-env": "^7.16.4",
     "babel-eslint": "^10.0.3",
```

Comment:
While we're making these updates, can we replace `babel-eslint` with `@babel/eslint-parser`, and `gulp-eslint` with `gulp-eslint-new`?  (ex: https://github.com/UCF/CM-WP-Theme-Template/commit/35b4da9d6d8bcca30b1ee897c2a2c6f7c775f166)

Reply:
Done: https://github.com/UCF/Pegasus-Theme/pull/53/commits/623ab6b90a5ec12f291d60a5c3b22aa519bd915a thanks!